app/views/abuse_reports/new.html.erb

```diff
@@ -8,49 +8,77 @@
 
 <!--main content-->
 <div class="userstuff">
-  <p><%= t(".purview.about",
-            tos_link: (link_to t(".purview.tos"), tos_path),
-            support_link: (link_to t(".purview.support"), new_feedback_report_path)).html_safe %></p>
-  <p><%= t(".do_not_spam_html") %></p>
-  <p>
-    <%= t(".reportable.intro") %>
+  <p class="important"><%= t(".purview.about",
+              tos_link: (link_to t(".purview.tos"), tos_path, name: "reportwhat")).html_safe %>
   </p>
+
+  <p><%= t(".purview.technical_help",
+              support_link: (link_to t(".purview.support"), new_feedback_report_path)).html_safe %>
+  </p>
+
+  <p><%= t(".purview.dmca_takedown",
+              dmca_link: (link_to t(".purview.dmca"), dmca_path)).html_safe %>
+  </p>
+
+  <p><strong><%= 
+              link_to "", name: "reportwhy" %>
+            <%= t(".reportable.intro") %>
+  </strong></p>
+
   <ul>
+    <li><%= t(".reportable.violation",
+              tos_link: (link_to t(".purview.tos"), tos_path, name: "reportwhat")).html_safe %></li>
+    <li><%= t(".reportable.content") %></li>
+    <li><%= t(".reportable.harassment") %></li>
     <li><%= t(".reportable.hack") %></li>
-    <li><%= t(".reportable.tos") %></li>
-    <li><%= t(".reportable.dmca.claim", dmca_link:
-            (link_to t(".reportable.dmca.policy"), dmca_path)).html_safe %></li>
+    <li><%= t(".reportable.suspended") %></li>
+  </ul>
+
+  <p><strong><%= 
+              link_to "", name: "reporthow" %>
+            <%= t(".include.intro") %>
+  </strong></p>
+
+  <ul>
+    <li><%= t(".include.username") %></li>
+    <li><%= t(".include.links") %></li>
+    <li><%= t(".include.quote",
+              tos_link: (link_to t(".include.ao3"), tos_path(anchor: 'content'))).html_safe %></li>
+    <li><%= t(".include.sources") %></li>
   </ul>
-  <p><%= t(".languages_html", list: @abuse_languages.map(&:name).to_sentence).html_safe %></p>
+
+  <p><%= t(".do_not_spam") %></p>
+
+  <p><strong><%= 
+              link_to "", name: "reportlanguage" %>
+              <%= t(".include.languages",
+                  list: @abuse_languages.map(&:name).to_sentence).html_safe %></strong>
```

Comment:
If you want to avoid this warning, `list_html` should remove the need for `.html_safe` (https://guides.rubyonrails.org/i18n.html#using-safe-html-translations)